src/main/java/serguei/http/HttpResponse.java

```diff
@@ -19,9 +19,15 @@
     private final long contentLength;
     private final boolean chunked;
 
-    public HttpResponse(InputStream inputStream) throws IOException {
+    public HttpResponse(InputStream inputStream, HttpRequestHeaders requestHeaders) throws IOException {
```

Comment:
It is a public class. Changing public constructor of a public class changes the library interface